src/routes/_layout.svelte

```diff
@@ -1,22 +1,21 @@
 <script>
-	import Nav from '../components/Nav.svelte';
-
-	export let segment;
+  import Nav from "../components/Nav.svelte";
+  export let segment;
 </script>
 
 <style>
-	main {
-		position: relative;
-		max-width: 56em;
-		background-color: white;
-		padding: 2em;
-		margin: 0 auto;
-		box-sizing: border-box;
-	}
+  /* main {
+    position: relative;
+    max-width: 56em;
+    background-color: white;
+    padding: 2em;
+    margin: 0 auto;
+    box-sizing: border-box;
+  } */
```

Comment:
Add a comment explaining why you're commenting this out so the next person (may be future you!) remembers